internal/metrics/chat_completion_metrics.go

```diff
@@ -126,11 +128,20 @@ func (c *chatCompletion) RecordTokenLatency(ctx context.Context, tokens uint32,
 
 	if !c.firstTokenSent {
 		c.firstTokenSent = true
-		c.metrics.firstTokenLatency.Record(ctx, time.Since(c.requestStart).Seconds(), metric.WithAttributes(attrs...))
+		c.timeToFirstToken = time.Since(c.requestStart).Seconds() * 1000
+		c.metrics.firstTokenLatency.Record(ctx, c.timeToFirstToken, metric.WithAttributes(attrs...))
 	} else if tokens > 0 {
 		// Calculate time between tokens.
-		itl := time.Since(c.lastTokenTime).Seconds() / float64(tokens)
-		c.metrics.outputTokenLatency.Record(ctx, itl, metric.WithAttributes(attrs...))
+		c.interTokenLatency = (time.Since(c.lastTokenTime).Seconds() * 1000) / float64(tokens)
+		c.metrics.outputTokenLatency.Record(ctx, c.interTokenLatency, metric.WithAttributes(attrs...))
 	}
 	c.lastTokenTime = time.Now()
 }
+
+func (c *chatCompletion) GetTimeToFirstToken() float64 {
```

Comment:
could you add the same style comment for interface implementation elsewhere:
```suggestion
// GetTimeToFirstToken implements [ChatCompletion.GetTimeToFirstToken].
func (c *chatCompletion) GetTimeToFirstToken() float64 {
```

Reply:
sure, will update this!